org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/services/ContentAssistFragmentTestLangGrammarAccess.java

```diff
@@ -450,11 +450,11 @@ public ParserRule getOpPostfixRule() {
 	}
 	
 	//XMemberFeatureCall XExpression:
-	//	XPrimaryExpression (=> ({XAssignment.assignable=current} ('.' | explicitStatic?="::")
-	//	feature=[types::JvmIdentifiableElement|FeatureCallID] OpSingleAssign) value=XAssignment
+	//	XPrimaryExpression (=> ({XAssignment.assignable=current} ('.' | explicitStatic?="::") feature=
```

Comment:
Still weird @ArneDeutsch 

@cdietrich Was this generated with the latest fixes applied?

Reply:
thought so. will check

Reply:
nope this linebreak withing assignment instead of before is still there

Reply:
Added a test. With latest conmmit the following:
```
XMemberFeatureCall returns XExpression:
	XPrimaryExpression
	(=>({XAssignment.assignable=current} ('.'|explicitStatic?="::") feature=[types::JvmIdentifiableElement|FeatureCallID] OpSingleAssign) value=XAssignment
	|=>({XMemberFeatureCall.memberCallTarget=current} ("."|nullSafe?="?."|explicitStatic?="::"))
		('<' typeArguments+=JvmArgumentTypeReference (',' typeArguments+=JvmArgumentTypeReference)* '>')?
		feature=[types::JvmIdentifiableElement|IdOrSuper] (
			=>explicitOperationCall?='('
				(
				    memberCallArguments+=XShortClosure
				  | memberCallArguments+=XExpression (',' memberCallArguments+=XExpression)*
				)?
			')')?
			memberCallArguments+=XClosure?
		)*;
```
will be formatted to:
```
XMemberFeatureCall returns XExpression:
	XPrimaryExpression
	(=>({XAssignment.assignable=current} ('.' | explicitStatic?="::")
	feature=[types::JvmIdentifiableElement|FeatureCallID] OpSingleAssign) value=XAssignment
	| =>({XMemberFeatureCall.memberCallTarget=current} ("." | nullSafe?="?." | explicitStatic?="::"))
	('<' typeArguments+=JvmArgumentTypeReference (',' typeArguments+=JvmArgumentTypeReference)* '>')?
	feature=[types::JvmIdentifiableElement|IdOrSuper] (=>explicitOperationCall?='('
	(memberCallArguments+=XShortClosure
	| memberCallArguments+=XExpression (',' memberCallArguments+=XExpression)*)?
	')')?
	memberCallArguments+=XClosure?)*;
```

This looks expectable for me.